src/server/api/routers/suggestions.ts

```
@@ -259,6 +258,71 @@ export const suggestionRouter = createTRPCRouter({
+              break
+            default:
+              notificationTitle = "Sugestão Atualizada"
+              const statusMapping = {
+                "NEW": "Nova",
+                "IN_REVIEW": "Em avaliação",
+                "APPROVED": "Aprovada",
+                "IN_PROGRESS": "Em execução",
+                "DONE": "Concluída",
+                "NOT_IMPLEMENTED": "Não implementada"
+              }
+              notificationMessage = `A sugestão #${suggestionData.ideaNumber} foi atualizada para "${statusMapping[input.status] || input.status}".`
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)